### PR TITLE
📖 Style bento sidebar hero example

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: extensions/amp-sidebar/1.0/example/index.html [hunk 1]
```diff
@@ -1,53 +1,111 @@
...
+        <ul>
+          <li>Nav item A</li>
+          <li>Nav item B</li>
+          <li>Nav item C</li>
+          <li>Nav item D</li>
+        </ul>
+      </bento-sidebar>
+
+      <script>
+        (async () => {
```
Comment (kristoferbaxter) [inline extensions/amp-sidebar/1.0/example/index.html:100]: Nit: Formatting is a bit strange here.

**Review by alanorozco — APPROVED** [review]
Approved if you do some minor changes: [no extra `<head>`, compress with SVGOMG and prettier-format](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhAEQPIGEAqAmgAoCiABABYwC2ANgHwA6Uq19zU55bcAhgBNO3bqhpwYfcmEp8ATgGcJAXiYgArjABmAWgAca8gHphIsRKlQ+41SABuASzgB3AA4Q5MQ5FgIYt5wcBGEplAThHMDgdQODKQxMWER4YBxg6OAYAIz8IHQUguCz5VCNU9Myk5J4FMDkHVxhyBTkwW2oYVwUkIyMwASgAOmtXVzkIACs4MBhBjwBzIxzYCEGJhTUGUtr6xtNk1DoHKABrKuqROTg6WwUYAE8MhUo4CTVzi-IH1zhbeAAPGB9BQbEAfC6UK5adowTrdXr9IYjMaTaazBZGOwABiWuXyhWKch0AEZBljBmAQe8uJ9uIkaQcdg0vAzPnwFPcoGBwdUWm01B0uj0+gNhjRRuMpjM5nJFtjcSt8eFCSSyWtQTzklsjEy9jzUHdHpUoFkIAJ7uRgFUtNAYDotNYHHR7khyDo+KMMvl7nc4DQADTkABCR1OAFk+GAAMo++A0ABitsDAHIo3B5hA4OQAKoASWTgYAShBTTAIP6qgAJa52CQOMB8QMAQXqfDogYUfCgCnycHqWhTTc9WdwEDoHnIpBokwcBfIqfTmZzucn04ms4rNIXGazefIMZoproyYA3CximATvNxuooAJXQBiaZaYlaADMZ6gAF8WB8spoyy4K1WW4GIihONJ3VGfg5C7KJXSgaA4E-C4LyvG871dGBYO7Vx5D8FDqlNORwjkBCkMI5JTX+fJZAECBnHIqBkJ5SBxzI8gHwAFjAABWPgADYP1Y9RFA8V13GOeA5EokQtHHPgYFdep5moWTuFiEJXS4rFXH+dSqDgBxVKU8g314vSDJoeR5mOV0sXIYkrhoWSfygD5KAAJktHkbVge1HWdV1k1zXw5DnTtu17fsDL8u0CgALzgV03y4yzfNtMDjOoV0jwEAzQ2iF5stM0liT0AqJGk-I8LAY55ldHRyQs-TWLHcTOKxLFPKyN8AE4rJs44dCycdL3ySRPFdAB2NLWtZNy-zxAplXkABtCAfigABdHyQPINDrwgW9704rRzoMvCBAEeqdGOQrtN0+aLmsmjNMoFLPKe1zf1ZdQ6D2i4jjuCajSYlj9qum6oAa8gsUG2U7Lhn73NZI5AeqOKApoJ0XXnULpIirseyUGKMv8xLkscgT0v2rHnCMkzXQE3j4Z5QqdGKpnyE8gy2I67i+ME4TIcEaH5hGsaTldPRaeSNyLm2B4Mn2HgjBeQRVdQU1zVV0R-1haByCCWxNoQJUinkTYWANOx5nIf56G7GFOmFZx3cGZw3xlRYvq6nU7cMRwXCDCB-lsByHM8riea462aVQPCQnILQnRuNREOYwwBFsMNI68ri7GjyssQANSxBK1HpUQk8ocgc7UMM30cvRKHKuwdE8ys3yLhKaCxHReLbvQO67nvPL7gfpqL4fS4E7vK5AeltjthhFaWADoE4fVljLC3CWNhuQBWy25GJQwT9sFTqHj2ltc3rgTbUMBxyUferZAThbftx26GdgVYSuDdh7L2PsjB+xxAoQOIBmjYQgCcX4ahBb8SEkHJwzhQ7hzUJHcgvFvbTVwfg2+PBa4pzTrYAApJ5VKQtUEwKPk3XigwBJmS4oMGOnk9A8wEoMPQdBprMJ0AIgSYBBieR0GIsREjeJwwkdNOg0jpF8EGEwmROCB6DAIViBRwjyDCKESogxAllGqLhmYjRWiFHMOYWACRnkpGSLUeQTR-DmF6MEbogRvFbGSPEY4sxLiVEqNEfY+xTD7GWP0Z4txwiQmOMkZYwxwSHGhOkbIzRi9yCbUjGke4thmFVy-gHeYeszAb0NlAbe+1RBeQYAAOT4I4eYikHBGyjASEo6tPJVLvv9UpBwjj1MacbOM5AmylEGZqMwgyGl2BGX6YMEyHD9Omcs2Z8yaDkFwEslZogZnDLSAszAOypmlD6fqBUe8T6Eh6dUA0dRmQrIABTsk5GAcgTyACU5BlBr2ArSAF0hoB3GaB0uQPz64QDAOocQsBBgAEd1B9nuGmDIMwPBPOTA+a58hiTJk+QZQF3A+DOD4GkaQ6g7gQBoKQDIsKYAKE9i8KAmA4Cp2YgITFu88g4vCgSqZgKfAgo9A4CFJKyVNF5YMeYEghwOC+Z+AVALejNAkOQdQrgDoP3IJGVIwKlW0notC+lCKkVyBRdcNEGKsVmygO-PlcwuRHEvBCr5Py14irmFtBVBrPhGphX4U1yLUVWrkJih8L8IBv15fix1L96wnFdd835OrXAOApK-OAPrqkAq-J87N69dQwC1ksM09winsEYCAf0IBNp6u7MgUA8hxjOGIPhBtKA2ykp9NWkAWRYKXgkFGawcAAAyxw4DIAdHQJQNb+2RgQTAKMtV6rIGwkimtfocjXTgAIUdXZ5jqD4DKxMchrKwlXZ2zQEBe2VoAOqUEOQoWqcA0zdjSA4RwDxkDgBBL244pMYDEFgvMayU62yzpAOsf47SYYZAAIrqAgPAcDM64A1rwooPsP7K29rGFJO9QQQjID0FiDD4wlB3tgq4H9Yw4Ck1rL2xFyG4DAc2goH97IdDMV3bu3tVxEUOCuMB49YGkDTsg0oHGa65AbuPvVBDSGUPiYg+hkAkgsiEbiMgTyNbsJktDPMUcNAxMgHo7xXtlK4D4D4FkDjKm0M1rsEi0K4RYBRgeY0Jsd4ozK0nQ5pQX4vxAA)

File: extensions/amp-sidebar/1.0/example/index.html [hunk 2]
```diff
@@ -1,53 +1,111 @@
...
-        document.querySelector('#open-sidebar').onclick = () => api.open();
-      })();
-    </script>
-  </body>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>bento-sidebar</title>
+
+    <head>
+      <script src="https://cdn.ampproject.org/bento.js"></script>
```
Comment (kristoferbaxter) [inline extensions/amp-sidebar/1.0/example/index.html:9]: Intentionally dropping the module/nomodule pattern here?

**Comment by alanorozco** [comment]
Also, I think we should use `module/nomodule` versions of the scripts.

File: extensions/amp-sidebar/1.0/example/index.html [hunk 3]
```diff
@@ -1,53 +1,111 @@
...
+      color: #4c5a63;
+      cursor: pointer;
+      float: right;
+      width: 40px;
+      height: 35px;
+      margin: 0 1rem;
+    }
+
+    h2 {
+      font-family: 'Inter', sans-serif;
```
Comment (kristoferbaxter) [inline extensions/amp-sidebar/1.0/example/index.html:41]: Inter doesn't appear defined in this stylesheet or elsewhere on the document.

**Review by kristoferbaxter — CHANGES_REQUESTED** [review]
I think the 'Inter' change is likely unintended and the module/nomodule needs a comment!

File: extensions/amp-sidebar/1.0/example/index.html (hunk 2 above)
Comment (westonruter) [reply extensions/amp-sidebar/1.0/example/index.html:9]: cf. #36992 where it's now:

```html
<script type="module" src="https://cdn.ampproject.org/bento.mjs" crossorigin="anonymous"></script>
<script nomodule src="https://cdn.ampproject.org/bento.js" crossorigin="anonymous"></script>
<script type="module" src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.mjs" crossorigin="anonymous"></script>
<script nomodule src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.js" crossorigin="anonymous"></script>
<link rel="stylesheet" href="https://cdn.ampproject.org/v0/bento-sidebar-1.0.css" crossorigin="anonymous">
```

Comment (kristoferbaxter) [reply extensions/amp-sidebar/1.0/example/index.html:9]: For my own clarity, does this mean all these examples will use the pattern listed here automatically by some means or the example here should be updated with this syntax?

**Review by kristoferbaxter — APPROVED** [review]
Changing to approval to unblock.

File: extensions/amp-sidebar/1.0/example/index.html (hunk 2 above)
Comment (sebastianbenz) [reply extensions/amp-sidebar/1.0/example/index.html]: Updated. Examples are not updated automatically, and most hero examples still use the the old pattern. We need a Bento Optimizer...

File: extensions/amp-sidebar/1.0/example/index.html (hunk 3 above)
Comment (sebastianbenz) [reply extensions/amp-sidebar/1.0/example/index.html]: removed

File: extensions/amp-sidebar/1.0/example/index.html (hunk 1 above)
Comment (sebastianbenz) [reply extensions/amp-sidebar/1.0/example/index.html]: fixed by fixing html syntax error

**Pull request merged**